l10n_ch_pain_base/models/account_payment_line.py

```diff
@@ -16,5 +16,5 @@ class AccountPaymentLine(models.Model):
     def invoice_reference_type2communication_type(self):
         res = super(AccountPaymentLine, self).\
             invoice_reference_type2communication_type()
-        res.update({'bvr': 'bvr', 'qrr': 'qrr'})
+        res.update({'bvr': 'bvr', 'QRR': 'qrr'})
```

Comment:
Is it normal that `bvr` isn't capitalized as well ?

Reply:
yes it is inconsistent, QRR backported from more recent versions and this already merged so fewer changes in code

Reply:
> yes it is inconsistent, QRR backported from more recent versions and this already merged so fewer changes in code

Ok, could you please write a `TODO` comment then, explaining that it should be unified in the future?

I pre-approve.